src/Microsoft.IdentityModel.Protocols.WsFederation/WsFederationMessage.cs

```diff
@@ -251,11 +251,8 @@ public virtual string GetTokenUsingXmlReader()
 
             string token = null;
             using (var sr = new StringReader(Wresult))
-            using (var xmlReader = new XmlTextReader(sr) { DtdProcessing = DtdProcessing.Prohibit })
+            using (var xmlReader = new XmlTextReader(sr) { DtdProcessing = DtdProcessing.Prohibit, XmlResolver = null })
             {
-                if (xmlReader.Settings != null)
```

Comment:
what is the history on this?

Reply:
That's part of what was causing the warning.
Some of our older code for netstandard 1.4 would not allow XmlreaderSettings to be set, so we had to check for null.
However, XmlTextReader allows setting DtdProcessing.Prohibit the code could be removed.